### ISSUE TITLE
[coc.nvim]: Uncaught exception: spawn python2 ENOENT

### ISSUE THREAD

**Issue by DMalladi** [neoclide/coc.nvim#2653]
**What I am trying to do?**
Installed COC Plugin in my VIM 8.2version
![image](https://user-images.githubusercontent.com/41450470/100183324-3f830f00-2ea4-11eb-88ea-07e15605044e.png)

I am able to install the COC Plugin successfully, after that, I tried to install the COC extensions via using **:CocInstall** command
1. :CocInstall coc-json => successfully installed and working as per expectations.
2. :CocInstall coc-python => Here, where the issue raised, and not working as per the expectations. 
**After the installation, coc-nvim, throwed this error, Could anyone please help me on fixing this error.** 
![image](https://user-images.githubusercontent.com/41450470/100183485-adc7d180-2ea4-11eb-812f-e7200c2ff472.png)

**Below error raise, when writing some python code in my vim editor, could anyone please help me on fixing this error.**
![image](https://user-images.githubusercontent.com/41450470/100183491-b15b5880-2ea4-11eb-9af3-8ca463ba0e19.png)

**Below error raise, when I start typing "python code" in my editor** 
![image](https://user-images.githubusercontent.com/41450470/100183498-b4eedf80-2ea4-11eb-87fd-3810acf1769b.png)

**Few more details:** 
I don't have python2 on my machine, I do have python3 installed and it is python 3.7 version

I did below configuration setting
:CocCommand python.setInterpreter => ../path/python37.exe

**List of extensions installed**
![image](https://user-images.githubusercontent.com/41450470/100183519-c1733800-2ea4-11eb-96aa-1c687c513f68.png)

**Comment by chemzqm** [comment]
No idea, but consider try coc-pyright

**Comment by DMalladi** [comment]
Thank You for your time to review my issue and responding, I started using pyright, and it working without any issues. Once again Thank You.

**Comment by ayushxx7** [comment]
I have been having the same issue. 
```
Uncaught exception: spawn python2 ENOENT
```

It's worth noting that I didn't have this issue before. Although, I can't tell exactly when I started facing it :( I should've raised it then and there itself. It has been at least a month, maybe more, but I was getting by.
Is there a fix for this?
I tried pyright earlier. It doesn't the same feel to it, and I don't want to switch if there's a way to suppress the error!
